docs/src/main/sphinx/release/release-393.md

```diff
@@ -0,0 +1,116 @@
+# Release 393 (10 August 2022)
+
+## General
+
+* Add support for the SQL [Merge](/sql/merge) statement. ({issue}`7933`)
```

Comment:
nit: We've historically styled SQL keywords as ``MERGE``. I don't how to do that and also preserve a link though.

Reply:
Good callout, and it's one of those things that looks like it's going to be an issue, but the fix is humorously simple.